app/renderer/components/preferences/payment/ledgerRecovery.js

```diff
@@ -58,12 +58,18 @@ class LedgerRecoveryContent extends ImmutableComponent {
     }
   }
 
+  onCancelRecovery (status) {
+    this.onRecoveryOverlay(status)
+    this.clearRecoveryStatus(status)
```

Comment:
I think this call is not needed, because we already have it in `onRecoveryOverlay`